app/admin/views.py

```diff
@@ -159,3 +160,21 @@ def delete_user(user_id):
         db.session.commit()
         flash('Successfully deleted user %s.' % user.full_name(), 'success')
     return redirect(url_for('admin.registered_users'))
+
+
+@admin.route('/users/update-donor-level', methods=['POST'])
+@login_required
+@admin_required
+def update_donor_level():
+    user_id = request.form.get('user_id')
+    dl_name = request.form.get('donor_level')
+
+    user = User.query.filter_by(id=user_id).first()
```

Comment:
is `abort(404)` an appropriate response? it's what is done in several above examples.